ecpy_hqc_legacy/instruments/drivers/visa/anapico.py

```diff
@@ -0,0 +1,199 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2015-2017 by EcpyHqcLegacy Authors, see AUTHORS for more details.
+#
+# Distributed under the terms of the BSD license.
+#
+# The full license is in the file LICENCE, distributed with this software.
+# -----------------------------------------------------------------------------
+"""Drivers for Anapico SignalGenerator using VISA library.
+
+"""
+from __future__ import (division, unicode_literals, print_function,
+                        absolute_import)
+
+import re
+from textwrap import fill
+from inspect import cleandoc
+
+from visa import VisaTypeError
+
+from ..driver_tools import (InstrIOError, instrument_property,
+                            secure_communication)
+from ..visa_tools import VisaInstrument
+
+
+class Anapico(VisaInstrument):
+    """
+    Generic driver for Rohde and Schwarz SMB100A SignalGenerator,
```

Comment:
Please correct the dosctring (wrong instrument and should start on the first line).